### ISSUE TITLE
[Adjust Rule] SIM201 - SIM207

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#13]
## Desired change

Rules for reducing the usage of "not" should not be applied if an exception is raised if the condition applies.

## Explanation

The pattern usually is:

```
if not condition:
    raise Exception("condition is not fulfilled but needs to be")
```

## Example

```
if not (minimum < maximum):
    raise ValueError(f"minimum={minimum} needs to be smaller than maximum={maximum}")
```

**Comment by MartinThoma** [comment]
Blocked by https://github.com/MartinThoma/flake8-simplify/issues/21
